css/properties/cursor.json

```diff
@@ -603,9 +604,17 @@
               "safari_ios": {
                 "version_added": "1"
               },
-              "samsunginternet_android": {
-                "version_added": null
-              },
+              "samsunginternet_android": [
+                {
+                  "version_added": "10.0",
+                  "notes": "Chrome also continues to support the prefixed versions."
```

Comment:
Thinking we should just nuke this note, as it just restates what's already stated down below.

```suggestion
```